wled00/wled.h

```diff
@@ -403,6 +403,8 @@ WLED_GLOBAL uint16_t e131ProxyUniverse _INIT(0);                  // output this
 #endif
 WLED_GLOBAL uint16_t e131Universe _INIT(1);                       // settings for E1.31 (sACN) protocol (only DMX_MODE_MULTIPLE_* can span over consequtive universes)
 WLED_GLOBAL uint16_t e131Port _INIT(5568);                        // DMX in port. E1.31 default is 5568, Art-Net is 6454
+WLED_GLOBAL byte e131Priority _INIT(0);                           // E1.31 port priority (if != 0 priority handling is active)
```

Comment:
We could probably default this to 1 to be more standard compliant in the 99% of cases where the user doesn't care about priorities. If I understand correctly, in order to be fully compliant we'd need to do separate priority handling for each universe, which I would prefer not to expend memory on, given that priority handling is only required in the first place if the user doesn't follow best practices (one sender at a time)

Reply:
0 = priority handling OFF = as actually is, without priority handling. So default 0 should be good in 99%.

> If I understand correctly, in order to be fully compliant we'd need to do separate priority handling for each universe

The only thing missing is to handle HTP = [Highest Takes Precedence](https://wiki.openlighting.org/index.php/HTP) for same priority sources. This means, merging each address value of same priority sources by HTP algorithm. This would need a refactoring as pointed out [here](https://github.com/Aircoookie/WLED/pull/3052#issuecomment-1407229536). If standard compliant is really wanted, this shall be covered by another additional PR .. 

WLED "best practice": Nevertheless it's a common way for E1.31 gear to take over / switch controllers by priority.

With this PR users are able to switch E1.31 senders without flickering, when switching to higher priority senders. When switching to lower priority senders a gap of `timeout=3` seconds is implemented, as higher priority senders disappear. As pointed out [here](https://github.com/Aircoookie/WLED/pull/3052#issuecomment-1407713063), this 'step back' is no big issue. It could be discussed if default `timeout=3` is to long, smallest possible would be `1`. Actually I don't know if `1s` is enough, so I leave it at `3s` for now. This highest priority tracking timeout may be changed in future, if E1.31 priority users frequently report it is to long .. TBD

Reply:
> We could probably default this to 1 to be more standard compliant in the 99% of cases

Of cause we could also init default `e131Priority` with 1, so priority handling is always active. This is also good but may impact performance due to `highPriority` handling some what. I think performance drop due to handling as implemented should not be noticeable. Finally this is an opt in|out decision, as you like ..

Reply:
As of my latest test comment
> Additional I didn't recognize any performance loss due to WLED priority handling.

Defaulting to priority handling always ON and eventually removing UI config (as @RichardTea mentioned) is still an option. Doing so, none of the different use cases / users have to worry about at all. But at the cost of the ability to always skip packages lower UI config. And a the little performance loss due to priority handling .. TBD

@Aircoookie @blazoncek Please decide & resolve how to finally implement this.